conda_rpms/build_rpm_structure.py

```diff
@@ -143,11 +139,19 @@ def create_rpmbuild_for_tag(repo, tag_name, target, config):
     with open(spec_fname, 'r') as fh:
         env_spec = yaml.safe_load(fh).get('env', [])
     create_rpmbuild_for_env(manifest, target, config)
-    pkgs = [pkg for _, pkg in manifest]
+
+    index = conda.fetch.fetch_index(list(set([url for url, pkg in manifest])),
+                                    use_cache=False)
+    resolver = Resolve(index)
+    dists = ['::'.join(['/'.join(url.split('/')[:-1]), pkg]) for url, pkg in manifest]
```

Comment:
Not pleasant.
Effectively this takes 
`http://www-avd/conda/RedHat.x86_64/linux-64    iris-2.0a0-py27_0`
and turns it into
`http://www-avd/conda/RedHat.x86_64::iris-2.0a0-py27_0`
as needed by the resolver.